core/templates/pages/topic-editor-page/modal-templates/create-new-story-modal.template.html

```diff
@@ -1,100 +0,0 @@
-      </div>
-      <input class="form-control e2e-test-new-story-title-field required" ng-model="story.title" maxlength="<[MAX_CHARS_IN_STORY_TITLE]>" aria-label="Title" autofocus>
-      <div>
-        <span class="oppia-input-box-subtitle">
-          <em>
-            Story title should be at most <[MAX_CHARS_IN_STORY_TITLE]> characters.
-          </em>
-        </span>
-      </div>
-    </div>
-    <div class="story-input story-input-description">
-      <div class="story-input-header">
-        <span>Description*</span>
-      </div>
-      <textarea rows="3" cols="50" maxlength="<[MAX_CHARS_IN_STORY_DESCRIPTION]>" ng-model="story.description" class="form-control e2e-test-new-story-description-field required" aria-label="Description"></textarea>
-      <span class="oppia-input-box-subtitle">
-        <em>
-          Story description should be at most
-          <[MAX_CHARS_IN_STORY_DESCRIPTION]> characters.
-        </em>
-      </span>
-    </div>
-    <div class="story-input">
-      <div class="story-input-header">
-        <span>Url Fragment*</span>
-      </div>
-      <input class="form-control e2e-test-new-story-url-fragment-field" ng-model="story.urlFragment" maxlength="<[MAX_CHARS_IN_STORY_URL_FRAGMENT]>"
-             ng-class="{'is-invalid': storyUrlFragmentExists || !validUrlFragmentRegex.test(story.urlFragment)}"
-             ng-change="onStoryUrlFragmentChange()"
-             aria-label="Url fragment">
-      <div>
-        <span class="oppia-input-box-subtitle">
-          <em>
-            The story URL fragment is used to uniquely access the story viewer page. It should consist of one or more hyphen-separated words, all in lowercase, with at most <[MAX_CHARS_IN_STORY_URL_FRAGMENT]> characters in total and must be unique across the topic. Please use meaningful keywords, and avoid using words like "and", "of", or "the".
-            This story can be accessed at the following URL:<br>
-            <[hostname]>/learn/<[classroomUrlFragment]>/<[topicUrlFragment]>/story/<[story.urlFragment]>
-          </em>
-        </span>
-        <div ng-if="storyUrlFragmentExists" class="oppia-input-box-subtitle text-danger">
-          <em>
-            This story fragment already exists in this or another topic.
-          </em>
-        </div>
-      </div>
-    </div>
-    <div class="story-input story-input-thumbnail">
-      <div class="story-input-header">
-        <span>Thumbnail Image*</span>
-      </div>
-      <oppia-thumbnail-uploader [preview-title]="story.title"
-                                [aspect-ratio]="'4:3'"
-                                [disabled]="false"
-                                [use-local-storage]="true"
-                                [allowed-bg-colors]="allowedBgColors"
-                                [thumbnail-bg-color]="story.getThumbnailBgColor()"
-                                preview-description-bg-color="#2F6687"
-                                (image-save)="updateView()">
```

Comment:
Just wondering: Do we not need a handler for this event after migration (because there isn't one)? (The function calls `$applyAsync`, so do we not need a `detectChanges()` call in the migrated code? Asking because I'm not familiar. Thanks!)

Reply:
we are using this earlier because we are in a hybrid state so change detection is not working properly, now it is in angular2+ so we don't need this.